runtime/compiler/env/j9method.cpp

```diff
@@ -3025,11 +3025,11 @@ void TR_ResolvedJ9Method::construct()
 
    static X NativeLibrariesMethods[] =
       {
-#if JAVA_SPEC_VERSION >= 17
+#if (17 <= JAVA_SPEC_VERSION) && (JAVA_SPEC_VERSION <= 18)
       {x(TR::jdk_internal_loader_NativeLibraries_load, "load", "(Ljdk/internal/loader/NativeLibraries$NativeLibraryImpl;Ljava/lang/String;ZZZ)Z")},
-#else /* JAVA_SPEC_VERSION >= 17 */
+#else /* (17 <= JAVA_SPEC_VERSION) && (JAVA_SPEC_VERSION <= 18) */
       {x(TR::jdk_internal_loader_NativeLibraries_load, "load", "(Ljdk/internal/loader/NativeLibraries$NativeLibraryImpl;Ljava/lang/String;ZZ)Z")},
```

Comment:
Similar comment here, Java 8/11 don't have `jdk/internal/loader/NativeLibraries`, there is no need to include this entry for those levels.

Reply:
The entire declaration of `NativeLibrariesMethods` can be omitted for versions prior to java 15.